docs/src/main/paradox/server.md

```diff
@@ -52,7 +57,50 @@ protobuf {
 ```
 @@@
 
-TODO settings for code generation: scala/java, client/server/both
+Maven
+:   @@@vars
+```
+<project>
+  <modelVersion>4.0.0</modelVersion>
+  <name>Project name</name>
+  <groupId>com.example</groupId>
+  <artifactId>my-grpc-app</artifactId>
+  <version>0.1-SNAPSHOT</version>
+  <dependencies>
+    <dependency>
+      <groupId>com.lightbend.akka.grpc</groupId>
+      <artifactId>akka-grpc-runtime_2.12</artifactId>
+      <version>${akka.grpc.project.version}</version>
+    </dependency>
+    <!-- for loading of cert, issue #89 -->
+    <dependency>
+      <groupId>io.grpc</groupId>
+      <artifactId>grpc-testing</artifactId>
+      <version>${grpc.version}</version>
+    </dependency>
+  </dependencies>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>com.lightbend.akka.grpc</groupId>
+        <artifactId>akka-grpc-maven-plugin</artifactId>
+        <version>${akka.grpc.project.version}</version>
+      </plugin>
+    </plugins>
+  </build>
+  <properties>
+    <maven.compiler.source>1.8</maven.compiler.source>
+    <maven.compiler.target>1.8</maven.compiler.target>
+    <akka.grpc.project.version>$projectversion$</akka.grpc.project.version>
+    <grpc.version>$grpc.version$</grpc.version>
+    <project.encoding>UTF-8</project.encoding>
+  </properties>
+</project>
+```
+@@@
```

Comment:
We should include these from the tester projects instead of having the xml in the markdown.